astroid/nodes/node_classes.py

```diff
@@ -366,21 +366,19 @@ def get_children(self):
 class LookupMixIn:
     """Mixin to look up a name in the right scope."""
 
-    @lru_cache(maxsize=None)
-    def lookup(self, name):
+    @lru_cache(maxsize=None)  # pylint: disable=cache-max-size-none # noqa
```

Comment:
Don't we want a max here? `128` on the same `LookupMixIn` seems like it should fit most of our uses in `pylint`?